app/javascript/controllers/registration_item_row_controller.js

```diff
@@ -2,13 +2,18 @@ import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
   static targets = ['barcode', 'catalogRecordId', 'sourceId', 'label']
+  static values = {
+    csv: Boolean
+  }
```

Comment:
Both the registration item row and registration tabs controllers need to know if the CSV side of the form has been toggled so that the fields on the user-input side of the form won't be validated. (They're blank.)